### ISSUE TITLE
New Self Hosted Kasm installation

### ISSUE THREAD

**Issue by Dean52** [kasmtech/workspaces-issues#556]
I am trying to run a self-hosted instance of Kasm. I have it running on a Virtual Debian 12 Server.
I have 24 GB of Ram supplied
240 GB Drive Allocated

I am still getting No resources available for some reason.  I have one running agent that is working fine. 

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/0ae918bc-54f9-4de0-961f-3515984cbff8)

When I run the health check all comes back as intact. 

Any ideas as to what I'm missing would be great.

**Comment by Dean52** [comment]
It says 100% installed.

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/e473a2ca-1a20-47a0-986c-23b0261e256a)

I get this error when trying to run it...

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/696f6991-e927-462d-b35f-ece6806bf861)

**Comment by Dean52** [comment]
Agent is running...

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/109fb58c-58dc-4df3-8ba0-d11403433570)

Not showing info for server and pool here though..

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/03e4fa95-33ec-48bf-a824-d99e608dc72e)

**Comment by j-travis** [comment]
Please review this: https://kasmweb.com/docs/latest/guide/troubleshooting/no_resources_available.html#no-resources-available

**Comment by Dean52** [comment]
> Please review this: https://kasmweb.com/docs/latest/guide/troubleshooting/no_resources_available.html#no-resources-available

Thanks...I went through this document and tried most of the potential fixes. No changes.

**Comment by j-travis** [comment]
What do the logs say?

Also, when you entered a ticket here, it asks you for a number of items. These help us help you. Please provide them.
https://github.com/kasmtech/workspaces-issues/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBug%5D+-+Example

You can view the logs from the UI or run the following command then run a session. Paste the output

`sudo docker logs -f --tail 50 kasm_api`

**Comment by Dean52** [comment]
Thanks for your help. I ended up spinning up a new Ubuntu 22.4 server and installing Kasm. It is working OK for me now. I do notice that the server and pool information is not displayed in this view.

![image](https://github.com/kasmtech/workspaces-issues/assets/59402359/fa5e9d41-890e-4f8d-ae5b-fa660ef909f9)

How much of the info from the command you posted do I need to submit? What are the items you say I needed to post? I must have overlooked the request for specific info. Appreciate the help.
